### ISSUE TITLE
hex code for colors

### ISSUE THREAD

**Issue by mikev-cw** [RIP-Comm/sossoldi#86]
I noticed that color is an integers used as the index for a list of default colors. Wouldn't it be better to save the HEX code instead and convert it directly to the Color class? With the current solution, if we were to edit the list of colors, we'd have to change all indexes across the database.

(From a comment of @GBergatto on #85)

**Comment by lucaantonelli** [comment]
I would agree, but now that we also have dark mode maybe it's easier to manage both types of colors with int, choosing the list based on the theme. I don't know if there could be a better solution for now.

**Comment by MirkoRaimo** [comment]
In 'lib/constants/style.dart', I see that colors are defined using the HEX format.  
Can you specify where 'int' colors are still used, or maybe this request has already been handled (and is now deprecated)? :)
